### PR TITLE
Use Flutter SDK for both Flutter and Dart snippets

### PR DESCRIPTION
**Pull request by domesticmouse**
Re-built https://github.com/dart-lang/dart-services/pull/655 after tidyup

fixes: dart-lang/dart-pad#1714

### CODE REVIEW

File: lib/src/sdk_manager.dart [hunk 1]
```diff
@@ -12,24 +12,14 @@ import 'package:yaml/yaml.dart';
...
-  static void setFlutterSdk(FlutterSdk value) {
-    _flutterSdk = value;
-  }
-
-  // The installed Dart SDK.
-  static Sdk _sdk;
-
   // The installed Flutter SDK.
-  static FlutterSdk _flutterSdk;
+  static FlutterSdk _sdk;
```
Comment (johnpryan) [inline lib/src/sdk_manager.dart:22]: Since there's only one SDK now, is the `SdkManager` class still useful?

File: lib/src/sdk_manager.dart [hunk 2]
```diff
@@ -51,19 +41,6 @@ abstract class Sdk {
...
-  @override
-  Future<void> init() => Future.value();
-
-  @override
-  String get versionFull =>
-      File(path.join(sdkPath, 'version')).readAsStringSync().trim();
-
-  @override
-  String get sdkPath => path.dirname(path.dirname(Platform.resolvedExecutable));
-}
```
Comment (johnpryan) [inline lib/src/sdk_manager.dart:65]: Since there's only one implementation of `Sdk` now, we can probably remove it too.

**Review by johnpryan — APPROVED** [review]
looks good, just a few questions

File: lib/src/sdk_manager.dart (hunk 1 above)
Comment (domesticmouse) [reply lib/src/sdk_manager.dart:22]: Want to do me a favour and raise this as a tech debt item on the tracker and assign it to me? I doubt I'm going to have time to re-arrange the deck chairs before March 3rd when this PR is going to land.

File: lib/src/sdk_manager.dart (hunk 2 above)
Comment (domesticmouse) [reply lib/src/sdk_manager.dart:65]: Ditto

File: lib/src/sdk_manager.dart (hunk 1 above)
Comment (johnpryan) [reply lib/src/sdk_manager.dart:22]: 👍

Comment (johnpryan) [reply lib/src/sdk_manager.dart:22]: https://github.com/dart-lang/dart-pad/issues/1768

File: lib/src/sdk_manager.dart (hunk 2 above)
Comment (johnpryan) [reply lib/src/sdk_manager.dart:65]: Added this to https://github.com/dart-lang/dart-pad/issues/1768 as well

**Pull request merged**